doc/caches.rst

```diff
@@ -434,7 +434,7 @@ You can set the ``sources`` to an empty list, if you use an existing geopackage
 
 .. versionadded:: 1.10.0
```

Comment:
.. versionadded:: 1.11.0
      ``region_name``, ``endpoint_url`` and ``access_control_list``